astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -410,3 +410,30 @@ def test_icrs_altaz_moonish(testframe):
 
     #now check that the distance change is similar to earth radius
     assert 1000*u.km < np.abs(moonaa.distance - MOONDIST).to(u.au) < 7000*u.km
+
+
+@remote_data
+def test_gcrs_self_transform_closeby():
```

Comment:
this also needs the ``@pytest.mark.skipif(str('not HAS_SKYFIELD'))`` decorator (and `HAS_SKYFIELD`` needs to be in the namespace, although the easiest is to just do ``from .test_solar_system import HAS_SKYFIELD`` to not have to duplicate that)